spec/models/manageiq/providers/kubernetes/container_manager_spec.rb

```diff
@@ -47,4 +47,40 @@
       expect(job.zone).to eq("default")
     end
   end
+
+  context "Proxy Configuration" do
```

Comment:
@enoodle Right; and whatever kubeclient does with the proxy uri is not a concern of ManageIQ's (kubeclient should test that it passes to rest-client properly. rest-client surely tests that it integrates with net::http properly. net::http tests that it does the request properly). 

It's a question of test responsibility and scope. From our perspective we only care that Kubeclient gets what it needs to do it's job and do not need a full integration test with a live proxy to test all of these external dependencies.